src/cephadm/cephadm

```diff
@@ -2650,7 +2650,10 @@ def command_bootstrap():
 
         host = get_hostname()
         logger.info('Adding host %s...' % host)
-        cli(['orch', 'host', 'add', host])
+        try:
+            cli(['orch', 'host', 'add', host])
+        except RuntimeError as e:
+            raise Error('Failed to ssh into host %s as root. Please check troubleshooting ssh-errors section of ceph documentation for more info.' % (host))
```

Comment:
```suggestion
            raise Error('Unable to add host: %s' % (host))
```

I think this can fail for more reasons than just failure to ssh ..

Reply:
I think you're probably right about it possibly being non-ssh issues; I'll remove that part. Do you think the recommendation to check the troubleshooting docs should be taken out entirely, or just changed so it it doesn't mention ssh specifically?

Reply:
I'd think consulting the troubleshooting guide is a given for any error, not just this one, so I'd probably drop that part too ...

@sebastian-philipp wdyt?

Reply:
I think we can drop this sentence. I mean,  https://docs.ceph.com/docs/master/cephadm/troubleshooting/#ssh-errors is basically same as the error message of the exception.